src/tests/test_dte_data_models.py

```diff
@@ -738,42 +792,52 @@ def _set_obj_2(self) -> None:
         self.obj_2 = obj
 
     def test_create_new_empty_instance(self) -> None:
-        with self.assertRaises(TypeError):
+        with self.assertRaises(pydantic.ValidationError):
             DteXmlReferencia()
 
     def test_init_fail_numero_linea_ref_out_of_range(self) -> None:
         self._set_obj_1()
 
         obj = self.obj_1
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 obj,
                 numero_linea_ref=0,
             )
         self.assertEqual(
-            assert_raises_cm.exception.errors(),
+            assert_raises_cm.exception.errors(
+                include_context=False,
+                include_input=False,
+                include_url=False,
+            ),
             [
                 {
                     'loc': ('numero_linea_ref',),
-                    'msg': '("Value \'numero_linea_ref\' must be a value between 1 and 40", 0)',
+                    'msg': 'Value error, ("Value \'numero_linea_ref\' must be a value between 1 '
+                    'and 40", 0)',
                     'type': 'value_error',
-                }
+                },
             ],
         )
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 obj,
                 numero_linea_ref=41,
             )
         self.assertEqual(
-            assert_raises_cm.exception.errors(),
+            assert_raises_cm.exception.errors(
+                include_context=False,
+                include_input=False,
+                include_url=False,
+            ),
             [
                 {
                     'loc': ('numero_linea_ref',),
-                    'msg': '("Value \'numero_linea_ref\' must be a value between 1 and 40", 41)',
+                    'msg': 'Value error, ("Value \'numero_linea_ref\' must be a value between 1 '
+                    'and 40", 41)',
                     'type': 'value_error',
-                }
+                },
```

Comment:
Revert change to reduce diff size (unless the comma was added by Black).

```suggestion
                }
```